kernel/data-pipeline/dialect/opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/OpenGaussWALDumper.java

```diff
@@ -92,6 +93,7 @@ protected void runBlocking() {
                 break;
             } catch (final SQLException ex) {
                 int times = reconnectTimes.incrementAndGet();
+                Thread.sleep(Math.min(10 * 1000L, 1000L << times));
```

Comment:
The reconnect might not continue one by one, do we need to sleep more and more time?